src/frequenz/sdk/_internal/_channels.py

```diff
@@ -91,3 +91,14 @@ async def _run(self) -> None:
     async def stop(self) -> None:
         """Stop the cache."""
         await cancel_and_await(self._task)
+
+    def __repr__(self) -> str:
+        """Return a string representation of the cache."""
+        return (
+            f"<LatestValueCache latest_value={self._latest_value!r}, "
+            f"receiver={self._receiver!r}, unique_id={self._unique_id!r}>"
+        )
+
+    def __str__(self) -> str:
+        """Return a string representation of the cache."""
+        return str(self._latest_value)
```

Comment:
Maybe the docstring should say the latest value in the cache instead?

Reply:
OK, I tend to use just a generic docstring for `__str__` and `__repr__` but maybe in this case it is worth to be more explicit.